code/modules/research/designs/weapons.dm

```diff
@@ -94,6 +94,18 @@
 	locked = TRUE
 	req_lock_access = list(access_armory, access_weapons)
 
+/datum/design/ionrifle
+	name = "Ion Rifle"
+	desc = "A man portable anti-armor weapon designed to disable mechanical threats."
```

Comment:
This has no armor penetration, why have its description be anti-armor?

Or is this the description of the Ion Rifle?

Reply:
It's copypasted from the ion rifle.

Reply:
anti-armor as in anti-vehicle, not armor piercing, i presume

Reply:
armor as in mechas

Reply:
Huh